src/extracts.jl

```diff
@@ -0,0 +1,104 @@
+"""
+```julia
+function extract_download(
+    api::IPUMSAPI, 
+    extract_number::Int, 
+    collection::String; 
+    output_path::String = pwd(), 
+    codebook_name::String = nothing, 
+    table_data_name::String = nothing, 
+    gis_data_name::String = nothing, 
+    codebook::Bool = true, 
+    table_data::Bool = true, 
+    gis_data::Bool = true
+)
+```
+
+Download files associated with a given IPUMS data extract.
+
+### Arguments
+
+- `api::IPUMSAPI` -- An `IPUMSAPI` object to establish connection details.
+
+- `extract_number::Int` -- extract ID assigned to the IPUMS data extract.
+
+- `collection::String` -- What IPUMS collection to be queried for the extract (options could include `"nhgis"`, `"usa"`, etc. corresponding to IPUMS NHGIS or NHGIS USA databases).
```

Comment:
Should we include an actual list of the databases that we can access. I believe we can access "nhgis", "usa", and "cps" through the api right now.

Reply:
Oh I do with the `ipums_data_collections` function. Then I was thinking about adding this into the docs.